packages/generator/src/generator.ts

```diff
@@ -199,6 +203,21 @@ export abstract class Generator<
     if (codeFileExtensions.test(pathEnding)) {
       templatedFile = this.replaceConditionals(inputStr, templateValues, prettierOptions || {})
     }
+    // templatedFile.match
+    const subTemplateRegExp = new RegExp(/{?\/\* template: (.*) \*\/}?/)
```

Comment:
I think we should also support `// template: ...` in addition to `/* template: ... */`